pkg/apiserver/slowquery/queries.go

```diff
@@ -93,13 +93,13 @@ type SlowQuery struct {
 }
 
 type GetListRequest struct {
-	RangeBeginTs uint     `json:"rangeBeginTs" form:"rangeBeginTs"`
-	RangeEndTs   uint     `json:"rangeEndTs" form:"rangeEndTs"`
-	DB           []string `json:"db" form:"db"`
-	Limit        uint     `json:"limit" form:"limit"`
-	Text         string   `json:"text" form:"text"`
-	OrderBy      string   `json:"orderBy" form:"orderBy"`
-	IsDesc       bool     `json:"desc" form:"desc"`
+	BeginTime int      `json:"begin_time" form:"begin_time"`
```

Comment:
Looks like you lost changes from https://github.com/pingcap-incubator/tidb-dashboard/pull/793

Reply:
no it's intended, In order to be consistent with the naming in the front-end and statements module